src/app/components/hris/employees/view-employee/view-employee.component.css

```diff
@@ -12,7 +12,9 @@ main {
     border-color: #E8EFFF;
     overflow: hidden;
 }
-
+#mainFilterContainer{
+    width:800px;
```

Comment:
Investigate:
Using Relative Units (e.g., Percentages)
Using Media Queries
Using Flexbox or Grid Layout
Using CSS Variables (Custom Properties)

To make our layout more flexible and responsive, it is best practice to use relative units, such as percentages, or CSS features like media queries and Flexbox. This ensures that the design adapts to different screen sizes and devices dynamically.

Resolve once noted.

Reply:
resolved